.tools/upload_to_s3.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/python3
+
+import logging
+import boto3
+from botocore.exceptions import ClientError
+import os
+import glob
+import sys
+
+
+def upload_files(base_path, bucket):
```

Comment:
This is probably much easier to do with the cli rather than custom python, would consider doing that either `awscli` or `snowsql`